package.json

```diff
@@ -20,7 +20,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "apidoc": "apidoc -o apidoc-out -i src/controllers/",
     "integration": "integration all",
-    "bump": "version-bump"
+    "bump": "version-bump",
+    "drop-oracle-tables": "scripts/drop-oracle-tables.sh"
```

Comment:
You mean as opposed to inlining it? I made a separate file because it's easier to read that way.